locale/en_US/migrate_user.json

```diff
@@ -0,0 +1,22 @@
+{
+  "subject_migrate_user": {
```

Comment:
It's used to localize the subject of the email here: https://github.com/mozilla/webmaker-mailroom/blob/1080fa97753e5cfc055e37b674ad532f5cb89d7a/index.js#L62